src/demo.html

```diff
@@ -78,6 +78,7 @@ <h1>MicroPython-micro:bit simulator example embedding</h1>
               <option value="display">Display</option>
               <option value="music">Music</option>
               <option value="sensors">Sensors</option>
+              <option value="pin_logo">Pin logo</option>
```

Comment:
Please can we keep these alphabetical and amend the comment to say they should be? Aiming to reduce trivial conflicts.

Reply:
[`493e35f`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/4/commits/493e35fb61319d0b0050bd5bb78e6e6469407872).